<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWidgetClass</class>
 <widget class="QMainWindow" name="FindWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FindWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="editor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextString">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>23</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPreString">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>23</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>23</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
